src/sentry/search/snuba/executors.py

```diff
@@ -1113,6 +1116,125 @@ class InvalidQueryForExecutor(Exception):
     pass
 
 
+def get_basic_group_snuba_lookup(search_filter: SearchFilter, attr_entity: Entity) -> Condition:
+    """
+    Returns the basic lookup for a search filter.
+    """
+    return Condition(
+        Column(f"group_{search_filter.key.name}", attr_entity),
+        Op.IN,
+        search_filter.value.raw_value,
+    )
+
+
+def get_assigned(search_filter: SearchFilter, attr_entity: Entity) -> Condition:
+    """
+    Returns the assigned lookup for a search filter.
+    """
+    users = filter(lambda x: isinstance(x, RpcUser), search_filter.value.raw_value)
+    user_ids = [user.id for user in users]
+    teams = filter(lambda x: isinstance(x, Team), search_filter.value.raw_value)
+    team_ids = [team.id for team in teams]
+
+    conditions = []
+    if user_ids:
+        assigned_to_user = Condition(Column("assignee_user_id", attr_entity), Op.IN, user_ids)
+        conditions.append(assigned_to_user)
+
+    if team_ids:
+        assigned_to_team = Condition(Column("assignee_team_id", attr_entity), Op.IN, team_ids)
+        conditions.append(assigned_to_team)
+    # asking for unassigned issues
+    if None in search_filter.value.raw_value:
+        # neither assigned to team or user
+        assigned_to_none_user = Condition(Column("assignee_user_id", attr_entity), Op.IS_NULL, None)
+        assigned_to_none_team = Condition(Column("assignee_team_id", attr_entity), Op.IS_NULL, None)
+        conditions.append(
+            BooleanCondition(
+                op=BooleanOp.AND, conditions=[assigned_to_none_user, assigned_to_none_team]
+            )
+        )
+
```

Comment:
What should happen when there was no valid user, team, or _None_, i.e. `conditions == []`?

There is no _NoOpCondition_ that could be returned, so probably should just error out?

Reply:
@vartec we should do an OR condition with nothing which returns 0 results.